Computer Vision/Real-Time Object Tracker/main.py

```
@@ -0,0 +1,96 @@
+import cv2
+import numpy as np
+import time
+import torch
```

Ruff (0.11.9)

`torch` imported but unused

Remove unused import: `torch`

(F401)